package.json

```diff
@@ -47,14 +47,14 @@
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
-    "babel-eslint": "^7.2.0",
-    "babel-preset-latest": "^6.24.0",
+    "babel-eslint": "^8.0.1",
+    "babel-preset-env": "^1.6.0",
     "babel-preset-stage-0": "^6.24.1",
     "cross-env": "^5.0.5",
-    "eslint": "^3.19.0",
-    "eslint-config-airbnb-base": "^11.2.0",
+    "eslint": "^4.8.0",
+    "eslint-config-airbnb-base": "^12.0.2",
     "eslint-plugin-import": "^2.7.0",
-    "jest": "^20.0.4",
+    "jest": "^21.2.1",
     "jest-junit": "^3.1.0",
     "opn-cli": "^3.1.0",
     "prop-types": "^15.5.10",
```

Comment:
I think `prop-types` can go up to 15.6.0

Reply:
it also looks like `rimraf` could go up to 2.6.2

Reply:
Strange, but `yarn outdated` didn't report those for me. Oh well, updated them now.